packages/webview_flutter/lib/webview_flutter.dart

```diff
@@ -194,6 +191,9 @@ class WebViewController {
 
   final MethodChannel _channel;
 
+  /// cached web settings
```

Comment:
A few nits:
  1. Start sentences with a capital letter, end with a period.
  2. This doesn't need to be a Dartdoc as it's private.
  3. I'm not sure what cached web settings means in this context, I'd just say `// The current web settings.` or not comment at all (as this is private and I believe self explanatory)